app/Http/Controllers/Admin/OverviewController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Http\Controllers\Admin;
+
+use App\Classes\Pterodactyl\PterodactylClient;
+use App\Http\Controllers\Controller;
+use App\Models\Pterodactyl\Egg;
+use App\Models\Pterodactyl\Location;
+use App\Models\Pterodactyl\Nest;
+use App\Models\Pterodactyl\Node;
+use App\Models\Server;
+use App\Models\User;
+use App\Settings\GeneralSettings;
+use Illuminate\Support\Facades\Cache;
+
+class OverviewController extends Controller
+{
+    public const TTL = 86400;
+
+    public function index(PterodactylClient $pterodactylClient)
+    {
+        /** @var GeneralSettings $settings */
+        $settings = app(GeneralSettings::class);
```

Comment:
Why app? You already inject the settings in `index(PterodactylClient $pterodactylClient)`. Why not also inject general settings? you can also use the __contrusctor to init these settings, so all future methods in this class have access to it to, which is usually preffered